### ISSUE TITLE
Increase line spacing and line height on zoom 

### ISSUE THREAD

**Issue by Chatcola** [tanganellilore/visavail#6]
Hi, is it possible to increase the line spacing and the line height while zooming.

I'm actually displaying a lot of data at 1 of line height, so y_titles are unreadable.

I would like to make them readable when zooming. Is it possible ?

Thanks by advance.

![yTitles](https://user-images.githubusercontent.com/25823356/54543689-67676300-499e-11e9-95d2-1ad31fee5661.PNG)

**Comment by tanganellilore** [comment]
Hi @Chatcola ,

please riproduce this error/bug in a jsfiddle or similar.
For zoom and multiple items I don't find any problem, as you can see [this](https://tanganellilore.github.io/visavail/docs/samples/zoom_enabled.html)

Thanks

**Comment by Chatcola** [comment]
Thanks for replying so fast, that you be a bit tricky to reproduce, data coming from an MySQL db and such stuff.
By the way, on your example, when you zoom, line height and line spacing remains the same.
I would like to know if it's possible to modify line height and line spacing on zoom.
I have to much data to display and y_titles are not readable obviously. 
Point would be to make y_title zoomed aswell, spaced while zooming.

**Comment by tanganellilore** [comment]
@Chatcola ,

at the moment there are two parameter for line space and line height, but it's not correlated by zoom.
This library for the moment is a library for timeline data and no a complex graph with zoom in a single point. 
You can introduce a pagination or similar for reduce the number of data in a single page (similar as visavail original) but in next month i think that introduce some example for pagination.

There are other library, similar to this, that do what you need but was develope in a different way and different scope.

I don't know if your request can be completed in a quick time or if is necessary for other people that use this library.

If you want can hel me in this task we can introduce, but it's not simple work with d3, svg and zooming.

good day

**Comment by Chatcola** [comment]
@tanganellilore 

Indeed, this Library looks incredibly great. The pagination is a good idea, I will make some tests.

I already tried a few Library. Anyone except Google Chart (working only through api) looks that great.

Don't have to worry about, it's a whim. By the way it's working great with 10 000+ rows.

Sorry I can't pull proper things. I'm not a JS developper, neither an web developper, just an data scientist having fun with cool js visualisations.

**Comment by tanganellilore** [comment]
@Chatcola ,

you have in mind something like [this](https://github.com/vasturiano/timelines-chart)???

If yes, I can try to work in this way, but it's not simple.

**Comment by Chatcola** [comment]
@tanganellilore 

I actually made a few prototypes with that one. Indeed a zoom like this would be awesome. I switched to visavail because of the lack of customisation.

It may be an great enhancement to this cool already enhanced fork.

I hope it won't be too hard to implement, neither time consuming.

**Comment by Chatcola** [comment]
@tanganellilore 

My prototype with [this](https://github.com/vasturiano/timelines-chartl)

![Timeline](https://user-images.githubusercontent.com/25823356/54611140-b2dd4800-4a56-11e9-831b-8219414526bf.PNG)

Looks identical but harder to configure and get exactly what you want. If I can help in some way ...

**Comment by tanganellilore** [comment]
@Chatcola,

your link is broken :-)

you can realize this with multiple line and mutiple series and custom categories.
The labrary work on timeline view and you can associate single block for a single line (i see in your picture)

At the moment we have'nt multiple axis and the graph move in x-axis with time and y axis with series/categories. In your example the graph move in x-axis with time, in y-axis with time and raggrupate externally with one categoriy.

It's completly different from actual situation

**Comment by Chatcola** [comment]
Damn, it was the same link obviously.
https://github.com/vasturiano/timelines-chart

Thanks, I didn't realise how those two graphs was different.

**Comment by tanganellilore** [comment]
So @Chatcola ,

just an [example](https://jsfiddle.net/1mpfo63a/).

This is not immediate but it's a simple workaround for you. With pagination you can easly implement if you have a lot of data.
Also with option you can reduce line height etc...

**Comment by Chatcola** [comment]
Thanks, I will try a few things.

Just hope it would be possible to have an select zoom some day.

Keep going, this is an awesome fork !
